### PR TITLE
add capture endpoint

### PR DESCRIPTION
**Pull request by SimonLab**
ref: https://github.com/dwyl/app/issues/234
capture

### CODE REVIEW

File: assets/css/app.css [hunk 1]
```diff
@@ -1,3 +1,3 @@
 /* This file is for your main application css. */
 
-@import "./phoenix.css";
+@import "./tachyons.css";
```
Comment (SimonLab) [inline assets/css/app.css]: see https://github.com/dwyl/app/issues/244

File: lib/app_web/router.ex [hunk 2]
```diff
@@ -25,6 +25,12 @@ defmodule AppWeb.Router do
     resources "/status", StatusController
     resources "/tags", TagController
     resources "/timers", TimerController
+
+    # capture
+    resources "/capture", CaptureController, only: [:new, :create]
```
Comment (nelsonic) [inline lib/app_web/router.ex]: thanks for adding this. 👍 
it has reminded me of what we want the **`default`** route to be ... 
https://github.com/dwyl/app/issues/246
(_detail will be added ahead of "Sprint 1" next week_)

File: priv/repo/migrations/20191115144039_update_item_text_type.exs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+defmodule App.Repo.Migrations.UpdateItemTextType do
+  use Ecto.Migration
+
+  def change do
+    alter table(:items) do
+      modify :text, :text
```
Comment (nelsonic) [inline priv/repo/migrations/20191115144039_update_item_text_type.exs]: https://github.com/dwyl/app/issues/234#issuecomment-554381219 👍

File: test/app_web/controllers/capture_controller_test.exs [hunk 4]
```diff
@@ -0,0 +1,26 @@
+defmodule AppWeb.CaptureControllerTest do
+  use AppWeb.ConnCase
+
+  @create_attrs %{text: "some text"}
+  @invalid_attrs %{text: nil}
+
+  describe "new capture" do
+    test "renders form", %{conn: conn} do
+      conn = get(conn, Routes.capture_path(conn, :new))
+      assert html_response(conn, 200) =~ "Capture"
```
Comment (nelsonic) [inline test/app_web/controllers/capture_controller_test.exs]: this test is fine for now. 
we will search for the placeholder text in a future iteration. 👍

File: test/app_web/controllers/page_controller_test.exs [hunk 5]
```diff
@@ -3,6 +3,6 @@ defmodule AppWeb.PageControllerTest do
 
   test "GET /", %{conn: conn} do
     conn = get(conn, "/")
-    assert html_response(conn, 200) =~ "Welcome to Phoenix!"
+    assert html_response(conn, 200) =~ "Empower people to maximise effectiveness, creativity and happiness"
```
Comment (nelsonic) [inline test/app_web/controllers/page_controller_test.exs]: 👍

**Pull request merged**